### ISSUE TITLE
install error for Java 0.9-10 using "renv" package management

### ISSUE THREAD

**Issue by fontbennz** [s-u/rJava#199]
Hi

I have installed rJava successfully in the past on both a Windows 7 machine and a RHEL7 server.
I am starting to use "renv" for package management, so created a renv project on the RHEL7 server. By version controlling the renv.lock file, I then pulled that to my project in my Windows 7 machine.
When I execute the renv::restore() command, it finds the older version of rJava from CRAN, but I receive errors when it tries to install.
FYI, I have set my environment variable for JAVA_HOME and I tried both the 32 bit and 64 bit versions of JRE.
Here is the error:

Error installing package 'rJava':

installing *source* package 'rJava' ...
** package 'rJava' successfully unpacked and MD5 sums checked
Generate Windows-specific files (src/jvm-w32) ...
make: Entering directory '/Users/user/AppData/Local/Temp/Rtmp4Qq41b/R.INSTALL31146163556d/rJava/src/jvm-w32'
C:/Rtools/mingw_64/bin/dlltool --as C:/Rtools/mingw_64/bin/as  --input-def jvm64.def --kill-at --dllname jvm.dll --output-lib libjvm.dll.a
C:/Rtools/mingw_64/bin/gcc  -O2 -c -o findjava.o findjava.c
C:/Rtools/mingw_64/bin/gcc  -s -o findjava.exe findjava.o
make: Leaving directory '/Users/user/AppData/Local/Temp/Rtmp4Qq41b/R.INSTALL31146163556d/rJava/src/jvm-w32'
Find Java...
  JAVA_HOME=C:/PROGRA2/Java/JRE181.0_2
=== Building JRI ===
  JAVA_HOME=C:/PROGRA2/Java/JRE181.0_2
  R_HOME=C:/PROGRA1/R/R-351.3
Creating Makefiles ...
Configuration done.
make -C src JRI.jar
make[1]: Entering directory '/Users/user/AppData/Local/Temp/Rtmp4Qq41b/R.INSTALL31146163556d/rJava/jri/src'
C:/PROGRA2/Java/JRE181.0_2/bin/javac  -d . ../RList.java ../RBool.java ../RVector.java ../RMainLoopCallbacks.java ../RConsoleOutputStream.java ../Mutex.java ../Rengine.java ../REXP.java ../RFactor.java ../package-info.java
sh: C:/PROGRA2/Java/JRE18~1.0_2/bin/javac: No such file or directory
make[1]: *** [Makefile.all:41: org/rosuda/JRI/Rengine.class] Error 127
make[1]: Leaving directory '/Users/user/AppData/Local/Temp/Rtmp4Qq41b/R.INSTALL31146163556d/rJava/jri/src'
make: *** [Makefile.all:19: src/JRI.jar] Error 2
**** WARNING: JRI could NOT be built
Set IGNORE=1 if you want to build rJava anyway.
ERROR: configuration failed for package 'rJava'
removing 'C:/Users/user/AppData/Local/Temp/RtmpA9HwAa/renv-tempdir-2f941c1ff28/renv-templib-2f941af368bc/rJava'
In R CMD INSTALL
Error: install of package 'rJava' failed

Thanks!

**Comment by fontbennz** [comment]
I found this:
https://github.com/s-u/rJava/issues/157
Seems like I have a similar case that "javac" not found (instead of "javah").

However, I am at a loss as to how to apply the fix, as during the install, the files appear to me to be stored in a temporary folder that then gets blitzed.

**Comment by fontbennz** [comment]
I tried installing OpenJDK versions 11 and 9.
OpenJDK had "javac", but then it failed on "javah"
I tried 9 because I found that from version 10 up, "javah was removed"
https://openjdk.java.net/jeps/313

All these attempts still failed.

So I ended up updating the version of rJava on the RHEL7 server and updating the "renv" lock file.
Pulling this to my Windows 7 machine means that I can then install the latest Windows binary of rJava and not have to build it from the tar ball.
